aiida/engine/transports.py

```diff
@@ -96,7 +97,7 @@ def do_open():
                         transport_request.future.set_result(transport)
 
             # Save the handle so that we can cancel the callback if the user no longer wants it
-            open_callback_handle = self._loop.call_later(safe_open_interval, do_open)
+            open_callback_handle = self._loop.call_later(safe_open_interval, do_open, context=contextvars.Context())
```

Comment:
Add a comment here about why it is (currently) necessary to set a specific context

Reply:
it also appears that context is not part of the `AbstractEventLoop` 🤔 :

> aiida/engine/transports.py:100: error: Unexpected keyword argument "context" for "call_later" of "AbstractEventLoop"  [call-arg]

you will need to add `type: ignore[call-arg]`